doc/nrf/known_issues.rst

```diff
@@ -2190,25 +2246,25 @@ NCSDK-13360: The :c:func:`nrf_recv` function crashes if closed by another thread
 Multiprotocol Service Layer (MPSL)
 ==================================
 
```

Comment:
v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
 
DRGN-18308: An assert could happen if there is more than one user of the scheduler
   Examples of users of the scheduler include bluetooth, 802.15.4, timeslot (e.g., flash driver).

v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
DRGN-18555: An assert could happen if requesting timeslots with type MPSL_TIMESLOT_REQ_TYPE_EARLIEST
   When requesting timeslots with type MPSL_TIMESLOT_REQ_TYPE_EARLIEST, an assert indicating that there is already an EARLIEST request pending in MPSL could occur.